lib/transport-common.js

```diff
@@ -47,22 +49,25 @@ const newConnectFn = (
       client.application = new Application('jstp', {});
     }
     if (!client.connectPolicy) {
-      client.connectPolicy = new SimpleConnectPolicy().connect;
+      client.connectPolicy = new SimpleConnectPolicy(client.session).connect;
```

Comment:
What is the use of the `client.session` parameter in `SimpleConnectPolicy`, ain't it better to just omit the parameters?